library/swift/src/GRPCRequestBuilder.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+/// Builder used for creating new gRPC `Request` instances.
+@objcMembers
+public final class GRPCRequestBuilder: NSObject {
```

Comment:
This class seems to almost exclusively delegate to `RequestBuilder`. Would it be worth either extending `RequestBuilder` (either via subclassing or class extension)?

Reply:
I had considered doing this, but the `GRPCRequestBuilder` actually exposes less than the existing `RequestBuilder`. I'd prefer to keep this minimal rather than adding gRPC-specific functionality to a more generic HTTP request builder